content/sensu-go/6.9/operations/manage-secrets/secrets-providers.md

```diff
@@ -274,11 +355,157 @@ name: vault
 {{< /code >}}
 {{< /language-toggle >}}
 
-### Spec attributes
+### `CyberArkProvider` spec attributes
+
+client       | 
+-------------|------ 
+description  | Map that includes [CyberArkProvider client attributes][20].
+required     | true
+type         | Map of key-value pairs
+example      | {{< language-toggle >}}
+{{< code yml >}}
+client:
+  account: sensu.io
+  appliance_url: http://localhost:8480
+  login: host/Sensu/sensuBackend
+  api_key: CONJUR_API_KEY
+  timeout: 1s
+  tls:
+    ca_cert: "/etc/ssl/certs/conjur_ca_cert.pem"
+  ttl: 60s
+{{< /code >}}
+{{< code json >}}
+{
+  "client": {
+    "account": "sensu.io",
+    "appliance_url": "http://localhost:8480",
+    "login": "host/Sensu/sensuBackend",
+    "api_key": "CONJUR_API_KEY",
+    "timeout": "1s",
+    "tls": {
+      "ca_cert": "/etc/ssl/certs/conjur_ca_cert.pem"
+    },
+    "ttl": "60s"
+  }
+}
+{{< /code >}}
+{{< /language-toggle >}}
+
+#### `CyberArkProvider` client attributes
+
+account      | 
+-------------|------ 
+description  | Conjur account name.
+required     | true
+type         | String
+example      | {{< language-toggle >}}
+{{< code yml >}}
+account: sensu.io
+{{< /code >}}
+{{< code json >}}
+{
+  "account": "sensu.io"
+}
+{{< /code >}}
+{{< /language-toggle >}}
+
+appliance_url | 
+--------------|------ 
+description   | Conjur appliance URL.
+required      | true
+type          | String
+example       | {{< language-toggle >}}
+{{< code yml >}}
+appliance_url: http://localhost:8480
+{{< /code >}}
+{{< code json >}}
+{
+  "appliance_url": "http://localhost:8480"
+}
+{{< /code >}}
+{{< /language-toggle >}}
+
+login        | 
+-------------|------ 
+description  | Conjur authentication login.
+required     | true
+type         | String
+example      | {{< language-toggle >}}
+{{< code yml >}}
+login: host/Sensu/sensuBackend
+{{< /code >}}
+{{< code json >}}
+{
+  "login": "host/Sensu/sensuBackend"
+}
+{{< /code >}}
+{{< /language-toggle >}}
+
+timeout      | 
+-------------|------ 
+description  | Provider connection timeout (hard stop).
+required     | false
+type         | String
+default      | 60s
+example      | {{< language-toggle >}}
+{{< code yml >}}
+timeout: 1s
+{{< /code >}}
+{{< code json >}}
+{
+  "timeout": "1s"
+}
+{{< /code >}}
+{{< /language-toggle >}}
+
+<a id="tls-conjur"></a>
+
+tls          | 
```

Comment:
This is entirely based on the VaultProvider tls attribute. Is this accurate for CyberArkProvider?

Also, do we have configuration information for tls with CyberArkProvider (like we do with [VaultProvider](https://docs.sensu.io/sensu-go/latest/operations/manage-secrets/secrets-management/#vault-tls-certificate-authentication))?

Reply:
CyberArk doesn't support TLS certificate authentication the way Vault does, so on the later question: no.

I reused the same TLS object from Vault for CyberArk. This looks good to me!